Atomic/scan.py

```diff
@@ -176,8 +176,9 @@ def gen_containers():
                                        if item['Id'] == self.get_input_id(scan_input)), None))
                 docker_object = docker_object if docker_object is not None \
                     else (next((item for item in images if item['Id'] == self.get_input_id(scan_input)), None))
-                docker_object['input'] = scan_input
-                scan_list.append(docker_object)
+                if docker_object is not None:
```

Comment:
I dont follow the logic here...what happens if docker_object IS none, should we throw an exception?